coordinator/tests/services/translation/response/release/helpers.ts

```diff
@@ -0,0 +1,252 @@
+import { fhir, hl7V3 } from "@models"
+import { LosslessNumber } from "lossless-json"
+
+interface GetExpectedMedicationRequestProps {
+	courseOfTherapyType: "acute" | "continuous" | "continuous-repeat-dispensing"
+	repeats?: {
+		high: number,
+		low: number
+	}
+}
+
+export const getExpectedMedicationRequest = (props: GetExpectedMedicationRequestProps): fhir.MedicationRequest => {
+	const { courseOfTherapyType, repeats } = props
```

Comment:
can do this as part of the function definition right?

Reply:
helpers.ts deleted